Assets/SolarSystem/Planets/LODs/Planet_LODs.mat

```diff
@@ -336,7 +336,7 @@ Material:
       second: {r: 1, g: 0, b: 0, a: 0}
     - first:
         name: _SunPosition
-      second: {r: -0.019818218, g: -0.09801179, b: 1.495989, a: 0}
```

Comment:
Thanks Tim, not sure why those were included. Do you have a preferred method to do this? (revert commit? some git magic I am not aware of?)